packages/builders/scripts/vats/terminate-governed-instance.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * @file Terminate price-feed governor instances such as mainnet v110.
+ * Functions as both an off-chain builder and an on-chain core-eval.
+ */
+
+/// <reference types="@agoric/vats/src/core/types-ambient"/>
+
+import { E } from '@endo/far';
```

Comment:
```suggestion
/**
 * @file Source for a core-eval to terminate governed contract instances along
 * with their governors by mapping a board ID to a key in the Bootstrap Powers
 * governedContractKits collection.
 * Also functions as an off-chain builder script for such core-evals:
 *     agoric run /path/to/$0 <$instanceHandleBoardID:$instanceKitLabel>...
 */

/// <reference types="@agoric/vats/src/core/types-ambient"/>

// dynamic import { makeHelpers } from '@agoric/deploy-script-support';
// dynamic import { getSpecifier } from '@agoric/internal/src/module-utils.js';
import { E } from '@endo/far';
```
And I think this file should move out of builders/scripts/vats for the same reason that add-governedContract.js moved to packages/governance/test/swingsetTests/contractGovernor/. Perhaps it should live in the cosmic-swingset package?